recipes/gpaw/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set build = 0 %}
+{% set name = "gpaw" %}
+{% set version = "1.5.1" %}
+{% set sha256 = "63a00bd97cc4e10fb5f5c718039af18c90c7e0e19897a6a852e5f54fc0c81776" %}
+{% set build_string = "py{}_{}_{}".format(environ.get('CONDA_PY', ''), mpi, build) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - 001_config-py.patch
+
+build:
+  number: {{ build }}
+  string: {{ build_string }}
```

Comment:
Why do you need this build string?

Reply:
I'm basically duplicating the recipe for LAMMPS which is also built with and without MPI:
https://github.com/conda-forge/lammps-feedstock/blob/master/recipe/meta.yaml
I just thought it was needed for the conda packages to have different names depending on MPI imlementation...